op-program/client/l2/engineapi/l2_engine_api.go

```diff
@@ -352,6 +357,36 @@ func (ea *L2EngineAPI) NewPayloadV3(ctx context.Context, params *eth.ExecutionPa
 	return ea.newPayload(ctx, params, versionedHashes, beaconRoot, nil)
 }
 
```

Comment:
This looks right to me but not sure if there is something else missing in here. cc @Inphi

Reply:
We also probably want to verify that requests is empty, but I'll do that in another PR.

Reply:
This looks good.